### ISSUE TITLE
[0.9.rc31] Some data not writing and leaving connection open…?

### ISSUE THREAD

**Issue by contentfree** [influxdb/influxdb#2666]
No error in the logs (STDERR and STDOUT both going into logfile). Log says `write series message published successfully for topic 4` after appending data for a number of points (sent in a batch) and `http` returns a `204` in response to the POST.

However, trying to select from the series results in just the columns in the output… no records. And it appears to leave network connections open in "ESTABLISHED" (according to netstat).

**Comment by contentfree** [comment]
I dropped all databases and recreated. No change in behavior.

**Comment by easyrasta** [comment]
Same problem for me. I have a single node, connection still established. Result is server doesnt write data and after some post I have:
```
broker cq: error hitting data node: http://host:8086: Post http://host:8086/data/process_continuous_queries: read tcp ipHost:808
6: use of closed network connection
```

**Comment by easyrasta** [comment]
Seem same problem as #2310

**Comment by eminden** [comment]
I guess what i am facing is same, I dig into this a bit, issue is getting solved at rc31 in case influx not receiving the tags section while writing to db, sending with the tags section causing influxdb to stop responding new writes.

**Comment by contentfree** [comment]
Downgrading to rc30 does *not* appear to be fixing the problem for me.

**Comment by pauldix** [comment]
We're reworking the entire write path so this should be solved in a release next week.

**Comment by kurtharriger** [comment]
I believe this happens on the creation of new tag columns.

I did first encounter this issue in rc-31 on empty database and had rolled back to rc-30 thinking it was completely broken, but due to issues in rc-30 gave rc-31 another try and things worked fine for few days.

Today, I modified our samza job to send tags instead of fields and each time it updated a series with new tags it crashed.  After the new tag columns were created things seem to be working normally again.
